windows/mdag_enable_disable/enable_mdag.yml

```diff
@@ -107,7 +111,8 @@
 
 - name: "Set the result of getting MDAG process"
```

Comment:
Task name can be changed to `Set the running status of MDAG process`

Reply:
Updated.